clients/cobol-lsp-vscode-extension/src/test/suite/index.ts

```diff
@@ -15,32 +15,34 @@
 import * as path from "path";
 import * as Mocha from "mocha";
 import { glob } from "glob";
+import type { Config, NYC } from "nyc";
 
 export async function run(): Promise<void> {
   const sourceRoot = path.join(__dirname, "..", "..");
 
   // initialize nyc code coverage
-  // eslint-disable-next-line @typescript-eslint/no-require-imports
-  const NYC = require("nyc");
-  const nyc = new NYC({
+  const NYC = (await import("nyc")) as new (cfg: Config) => NYC;
```

Comment:
We could try moving to `c8` here as well, to get rid of all of this.

Reply:
I would do that in separate PR.